src/Chapter1.hs

```diff
@@ -640,8 +657,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | abs(n) < 10 = abs(n)
+    | otherwise = firstDigit(div (abs(n)) 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼 

And also, you usually separate function and arguments with spaces:
```suggestion
    | abs n  < 10 = abs n
    | otherwise = firstDigit (div (abs n) 10)
```